drupal/web/modules/custom/wunder_democontent/sources/users.csv

```diff
@@ -0,0 +1,3 @@
+id,name,mail,password,roles
```

Comment:
these test users will be created automatically and can be used to test the login